### DISCUSSION TITLE
Is there a way to add notice box for important messages on index page?

### DISCUSSION THREAD

**Discussion by jakesullyneytiri** [mastodon/mastodon#27933] [Q&A]
Is there a way for me to add a important notice message on index page / about page where you signup/login so users knows about important information?

Since i would like to add about some e-mail services aren't accepting mails from mailgun for some strange reason.

If it is please tell me code and in which file to edit and where file is so i can add this information and which line in file to put code at.

It can be a notice box at top of the page that is centered or goes from left of web browser to right and fills whole area above  and pushes stuff down bellow downwards a bit. So users sees the important information.

/Thanks

**Comment by ClearlyClaire** [comment]
There is no feature to do that, but you could have a look at `app/views/about/show.html.haml` to add it yourself.

**Comment by jakesullyneytiri** [comment]
> There is no feature to do that, but you could have a look at `app/views/about/show.html.haml` to add it yourself.

I know there is no feature for it, but i need some 1 who can give me a code and css code that i can add to custom css thingy in admin area and then where to put html code. 

So i have a notice box for important messages when needed. Also i need to know where in show.html.haml file it has to be to get it on top of page.
